public/css/stylesheet.css

```diff
@@ -0,0 +1,20 @@
+body {
```

Comment:
Watch your indentation file - its different for the first set of styles and the second set of styles.